### ISSUE TITLE
transform  not work

### ISSUE THREAD

**Issue by Aquarius-Sally** [bokuweb/react-rnd#421]
### Overview of the problem

I'm using react-rnd **version** [x.x.x]   
      
My **browser** is:   
   
I am sure this issue is **not a duplicate**?
   
### Reproduced project

https://codesandbox.io/s/y3997qply9

### Steps to Reproduce

1. First Step
2. Second Step
3. and so on...

**Comment by Aquarius-Sally** [comment]
react-rnd : ^7.4.1,
transform: rotate(30deg)

**Comment by dholakiyaankit** [comment]
Yes, its been overwritten by draggable, we need some updates on this.

**Comment by rullymartanto** [comment]
any update on this ?
